### DISCUSSION TITLE
Downloading the 22.0 Version of STP

### DISCUSSION THREAD

**Discussion by lectsuliv** [SimDynamX/SpaceTeamsPRO#25] [General]
I was able to download the STP software on my personal computer, but when I try to run it, I keep getting an error message as follows.

2024-05-04T23:14:48 [ERROR]--[PySys_ParamWrangler->Server @NID:1]: Could not connect to server: [10061] No connection could be made because the target machine actively refused it.

I'm not entirely sure on what the issue is, if it's an internal or external issue. Is there anything I can do to make the software run properly?

**Comment by connorjak** [comment]
I'm guessing the `FirstTimeSetup.bat` script hasn't been ran yet? That would mean ParamWrangler Python System is crashing on startup (can see if this is the case if you open the `Sim_(SIM_CONFIG_NAME)_PySys_ParamWrangler.log` file).

Running `SpaceCRAFT/FirstTimeSetup.bat` would fix this if this is the issue.

If you post your `Sim_(SIM_CONFIG_NAME).log` file (from `Compute_Server/Logs`), that will help with diagnosing the problem.
